sqlglot/dialects/duckdb.py

```diff
@@ -92,6 +92,14 @@ def _timediff_sql(self: DuckDB.Generator, expression: exp.TimeDiff) -> str:
     return self.func("DATE_DIFF", unit_to_str(expression), expr, this)
 
 
+# BigQuery -> DuckDB conversion for the DATETIME_TRUNC function
+def _timestamp_trunc(self: DuckDB.Generator, expression: exp.DatetimeTrunc) -> str:
+    time_cast = exp.cast(expression.this, exp.DataType.Type.DATETIME)
+
+    args = [unit_to_str(expression), time_cast]
+    return self.func("DATE_TRUNC", *args)
+
+
```

Comment:
We can remove this altogether and implement it inline in `TRANSFORMS`, see other comment.
```suggestion
```

Reply:
done